themes/godotengine/pages/education.htm

```diff
@@ -0,0 +1,82 @@
+title = "Education"
+url = "/education"
+layout = "default"
+is_hidden = 0
+==
+{##}
+{% put head_text %}
+<p class="small">
+  Godot is already being used in many classrooms across the world. If you are interested in including it in your
+  curriculum, you can find answers to some of the most common questions on this page.
+</p>
+{% endput %}
+
+<style>
+  .question h3 {
+    margin: 2em 0 0;
+  }
+
+  .question p {
+    margin-top: 0.5em;
+  }
+</style>
+
+<div class="question">
+  <h3>Can I teach Godot in my classroom?</h3>
+  <p>Godot&nbsp;Engine is <a href="https://en.wikipedia.org/wiki/Free_and_open_source_software">free and open source
+      software</a> released under the permissive <a href="/license">MIT license</a>.<br>
+    You can use Godot in your classroom without any restrictions and with zero licensing
+    costs.
+  </p>
+  <p>You can read more about <a href="/license">our license here</a>.</p>
+</div>
+
+<div class="question">
+  <h3>Will the Godot project sign a services agreement provided by my institution?</h3>
+  <p>The Godot Engine project is unable to accept any terms and conditions imposed by organizations, nor will it
+    provide warranties or representations about the Godot Engine. This is clearly outlined in the text of the <a
+      href="/license">MIT license</a>.</p>
+  <p>As a free and open source project, Godot is not backed by a corporation or any individual. Accordingly, it doesn't
+    have the legal capacity to enter into agreements, even if such agreements did not conflict with the MIT license.</p>
+</div>
+
+<div class="question">
+  <h3>What data does the Godot Engine executable collect from users?</h3>
```

Comment:
```suggestion
  <h3>What data does the Godot Engine collect from users?</h3>
```

Simplify this as the target audience may include more non-tech users than usual (whoever is responsible for legal vetting at a edu institution)?